web/src/Setup.tsx

```diff
@@ -39,389 +34,114 @@ const SSH_AUTH_KEYS_EXAMPLE =
-                    <li>Always get the newest software for your LXA TAC</li>
-                    <li>Easier to set up</li>
-                  </ul>
+                title: "Add SSH keys",
+                description:
+                  "Deploy SSH keys on your LXA TAC so you can log into it",
+                content: (
+                  <Container>
+                    <ConfigEditor
+                      path="/v1/tac/ssh/authorized_keys"
+                      language="text"
+                      defaultContent={SSH_AUTH_KEYS_EXAMPLE}
+                    />
+                  </Container>
                 ),
-                drawbacks: (
-                  <ul>
-                    <li>Can get tedious in larger fleets</li>
-                  </ul>
+              },
+              {
+                title: "Configure Labgrid",
+                description: "Configure your labgrid Exporter",
+                isOptional: true,
+                content: (
+                  <Container>
+                    <LabgridConfig />
+                  </Container>
                 ),
               },
               {
-                name: "Setup via custom RAUC bundle",
-                kind: WizardChoice.CustomBundle,
-                benefits: (
-                  <ul>
-                    <li>
-                      Quickly integrate new LXA TACs into an existing fleet
-                    </li>
-                    <li>Deploy a custom selection of software</li>
-                    <li>Deploy custom config</li>
-                  </ul>
+                title: "Test Labgrid",
+                description:
+                  "Make sure your labgrid Exporter Service looks healty",
+                isOptional: true,
+                content: (
+                  <Container>
+                    <LabgridService />
+                  </Container>
                 ),
-                drawbacks: (
-                  <ul>
-                    <li>
-                      Requires up-front work to configure and build bundles
-                    </li>
-                    <li>
-                      You have to manually re-build bundles to get software
-                      updates
-                    </li>
-                  </ul>
+              },
+              {
+                title: "Complete Setup",
+                description: "Make sure everything is working alright",
+                content: (
+                  <Container>
+                    <Box>You are about to complete the setup wizard.</Box>
+                    <Box>
+                      You will not be able to re-enter the setup wizard to
+                      deploy new SSH keys via the web interface. You can however
+                      re-enable the setup wizard using the buttons and the
+                      screen on the device. If you do not have physical access
+                      the TAC you should make sure that you can log in to the
```

Comment:
If you do not have physical access _to_ the TAC, ...

Reply:
Fixed in #51 and trickled down to here. Thanks.